proposals/low-level-struct-improvements.md

```diff
@@ -299,6 +301,12 @@ Further treating the input to an `out` parameter as returnable is extremely conf
 
 This means going forward `out` will match developers intuition when it comes to ref safety: `out` means `out`.
 
+<a name="implicitly-scoped"></a>
+Overall there are three `ref` location which are implicitly declared as `scoped`:
+- `this` on a `struct` instance method
+- `out` parameters 
+- `ref` parameters which refer to a `ref struct`
```

Comment:
This default flip is easy to add to rules and fixes the last usability hurdle we had with ref fields. There are other alternatives we are exploring that would remove this special casing. Short term though wanted to focus on getting the rules tightened up as teams are locking down on implementations. 

Next week we'll be exploing the other options to see if we can remove this special case.